desktop/src/renderer/src/components/file-list/FolderDisplayGrid.tsx

```diff
@@ -85,7 +87,7 @@ export const FolderDisplayGrid = ({ rows, onFolderDelete }: Props) => {
 						{params.row.progress === 100 ? (
 							<CheckIcon color="success" />
 						) : (
-							<LoadingIcon sx={{ color: "#f7d219" }} />
+							<LoadingIcon style={{ color: `${theme.palette.warning.light}` }} />
```

Comment:
This should be main, not light

Reply:
I liked the look of the lighter colour more, but again all these things may change with wireframes.